resources/views/profile/profile.twig

```diff
@@ -75,9 +75,34 @@
                                             <p class="mb-0 mr-4">
                                                 {{ trans('visiosoft.module.profile::message.disable_account') }}
                                             </p>
-                                            <a href="{{ url_route("visiosoft.module.profile::profile_close_account") }}" class="btn btn-danger account-freeze text-white">
+                                            <a type="button" data-toggle="modal" data-target="#freezeModal" href="" class="btn btn-danger account-freeze text-white">
```

Comment:
href is can not null. this is not good for seo. replace to button

Reply:
Yes sir resolved ...